packages/tests/src/_jest/fixtures/hard_project/package.json

```diff
@@ -6,10 +6,5 @@
     "packageManager": "yarn@4.2.1",
     "dependencies": {
         "chalk": "2.3.1"
-    },
-    "devDependencies": {
-        "react": "19.0.0",
-        "react-dom": "19.0.0",
-        "react-router-dom": "6.28.0"
```

Comment:
Not sure why it was there, probably some legacy left overs from the rum plugin's implementation.